compiler/src/dmd/doc.d

```diff
@@ -5180,10 +5181,10 @@ private void highlightCode3(Scope* sc, ref OutBuffer buf, const(char)* p, const(
  */
 private void highlightCode2(Scope* sc, Dsymbols* a, ref OutBuffer buf, size_t offset)
 {
-    uint errorsave = global.startGagging();
```

Comment:
Nice

Reply:
Yeah, gagging is an ugly kludge. ErrorSink is a solid solution.